projects/container_bench/testing/exec_time.py

```
@@ -5,56 +5,167 @@
+                universal_newlines=True
+            )
+            if process_result.returncode == 0:
+                watts = self._darwin_parse_power_output(process_result.stdout)
+            else:
+                print("Error occurred while executing powermetrics.")
+                print(f"Return code: {process_result.returncode}")
+                print(f"Error output: {process_result.stderr}")
+        except subprocess.TimeoutExpired:
+            print("Error: powermetrics command timed out.")
+        finally:
+            return watts
```

Ruff (0.12.2)

`return` inside `finally` blocks cause exceptions to be silenced

(B012)